nmma/em/likelihood.py

```diff
@@ -97,17 +99,13 @@ def __init__(
         self.verbose = verbose
 
     def __repr__(self):
-        return self.__class__.__name__ + "(light_curve_model={},\n\tfilters={}".format(
-            self.light_curve_model, self.filters
-        )
+        return self.__class__.__name__ + "(light_curve_model={},\n\tfilters={}".format(self.light_curve_model, self.filters)
```

Comment:
again, the character limit

Reply:
https://github.com/nuclear-multimessenger-astronomy/nmma/pull/253#discussion_r1347373764